### PR TITLE
Add a whole bunch of tests

### PR DESCRIPTION
**Pull request by mcnuttandrew**
This adds tests for @jckr's color article, and a couple of other small things

### CODE REVIEW

**Comment by mcnuttandrew** [comment]
(the 3% drop in coverage is a lie: I dropped the test files from the coverage count, bc that felt like cheating)

File: tests/components/treemap-tests.js [hunk 1]
```diff
@@ -69,3 +72,21 @@ test('Treemap: Empty treemap', t => {
 
   t.end();
 });
+
+test('Treemap: SimpleTreemap', t => {
+  const $ = mount(<SimpleTreemap />);
+  t.equal($.find('.rv-treemap__leaf').length, 252, 'should find the right number of children');
+  t.equal($.find('.rv-treemap').text(), '', 'should find the correct text shown');
+
+  $.find('.showcase-button').at(1).simulate('click');
```
Comment (Apercu) [inline tests/components/treemap-tests.js]: what is the purpose of that simulation?

Comment (mcnuttandrew) [inline tests/components/treemap-tests.js]: thats left over from something that didn't work, good catch

**Comment by jckr** [comment]
that's great that you wrote those as I was about to start doing that

**Comment by mcnuttandrew** [comment]
> that's great that you wrote those as I was about to start doing that

whoops! sorry, i thought part of the fast merge of your colors PR was that i'd write tests for you. my bad

**Comment by jckr** [comment]
ha don't let me stop you. i was actually wondering what to test in examples besides them mounting.

**Pull request merged**